vigetlabs/storyboard
# [#81] Allow user to upload pictures to scenes

### What it does

Allow user to add an image to each of their scenes. 

### How to test

- Create a new adventure or edit an existing adventure
- Create a new scene or select an existing scene
- In the Images section of the editor, click Choose File to upload an image
- Confirm image displays in editor
- Click Play and navigate to your scene with the image
- Confirm image displays on screen

### Screenshots

<img width="1178" alt="Screen Shot 2021-07-29 at 11 16 20 AM" src="https://user-images.githubusercontent.com/25255820/127518501-2647374b-e75b-48da-95e6-35afee5bea86.png">

<img width="437" alt="Screen Shot 2021-07-29 at 11 17 53 AM" src="https://user-images.githubusercontent.com/25255820/127518747-80e88b44-e524-4153-ae19-0e6483fcb221.png">
